### PR TITLE
Universal SQL: Fix `build:sources` regressions

### PR DESCRIPTION
**Pull request by ItsMeBrianD**
### Description

[asciicast](https://asciinema.org/a/Ky1tnnKsCgdadbaZYhZ0lxMWt)

### Checklist
- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)

### CODE REVIEW

File: packages/plugin-connector/src/data-sources/build-sources.js [hunk 1]
```diff
@@ -93,14 +93,18 @@ export const buildSources = async (
 			},
 			/**
 			 * @param {string} name
+			 * @returns {boolean} true if query is included in filters
 			 */
 			isFiltered: (name) =>
-				Boolean(filters?.queries?.has(name) || filters?.queries?.has(`${source.name}.${name}`)),
+				Boolean(filters?.queries?.has(name) || filters?.queries?.has(`${source.name}.${name}`)) ||
+				!filters?.queries,
```
Comment (csjh) [inline packages/plugin-connector/src/data-sources/build-sources.js:100]: Doesn't this make the `Boolean` part irrelevant? If `filters?.queries` is true, then it doesn't matter if either call of `filters?.queries?.has`  is true

Comment (ItsMeBrianD) [reply packages/plugin-connector/src/data-sources/build-sources.js:100]: This should mean:

```
table name generically referenced || table name specifically referenced || no filters were specified

```

Comment (csjh) [reply packages/plugin-connector/src/data-sources/build-sources.js:106]: Ah yeah missed the exclamation mark

File: sites/example-project/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	"scripts": {
 		"dev": "vite dev --port 3000",
 		"build": "vite build",
-		"sources": "evidence sources",
+		"build:sources": "__EXAMPLE_PROJECT=true evidence sources",
```
Comment (csjh) [inline sites/example-project/package.json:7]:
```suggestion
		"sources": "__EXAMPLE_PROJECT=true evidence sources",
```

**Review by csjh — APPROVED** [review]

**Pull request merged**